engine/src/test/java/io/zeebe/engine/util/StreamProcessorRule.java

```diff
@@ -295,15 +309,18 @@ private static String getLogName(final int partitionId) {
 
     @Override
     protected void before() {
-      streams =
-          new TestStreams(
-              tempFolder, closeables, serviceContainerRule.get(), actorSchedulerRule.get());
+      streams = new TestStreams(tempFolder, closeables, actorSchedulerRule.get());
 
       int partitionId = startPartitionId;
       for (int i = 0; i < partitionCount; i++) {
         streams.createLogStream(getLogName(partitionId), partitionId++);
       }
     }
+
+    @Override
+    protected void after() {
+      streams = null;
```

Comment:
Should we not close the `TestStreams`? I see it manages a bunch of `closeables`.

Reply:
Closeables are managed outside